packages/site/src/ui/open-issue-prompt.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+
+import Button from '@pluralsight/ps-design-system-button/react'
+
+import { GithubIcon } from '.'
+
+class OpenIssuePrompt extends React.PureComponent {
+  render() {
+    return (
+      <div className="open-issue-prompt">
+        <Button
+          href="//github.com/pluralsight/design-system/issues/new"
+          icon={<GithubIcon />}
+          target="_blank"
+        >
+          Open an issue
+        </Button>
+
+        <style jsx>{`
+          .open-issue-prompt {
+            position: fixed;
+            bottom: 20px;
+            right: 20px;
+          }
+          .open-issue-prompt > :global(a) {
```

Comment:
This feels like a weakness of styled-jsx.

Reply:
definitely - i tried to work around it but came up short